src/main/kotlin/apply/application/ApplicationFormService.kt

```diff
@@ -60,4 +61,70 @@ class ApplicationFormService(
         )
         applicationFormRepository.saveAll(applicationForms)
     }
+
+    fun save(applicantId: Long, applicationFormSaveRequest: ApplicationFormSaveRequest) {
```

Comment:
변수 이름은 `applicationFormSaveRequest` 대신 `request`가 어떨까요?

```suggestion
    fun save(applicantId: Long, request: ApplicationFormSaveRequest) {
```